### ISSUE TITLE
Mongoose and Kurento-client conflict 

### ISSUE THREAD

**Issue by Steeew47** [Automattic/mongoose#8105]
**Do you want to request a *feature* or report a *bug*?**
*I want to report a bug*

**What is the current behavior?**
When you trying create kurento object using require("kurento-client") and mogose using require("mongoose") in the same file this cause an error.
```
/usr/lib/node_modules/mongoose/lib/error/validation.js:74
ValidationError.prototype.toJSON = function() {
                                 ^

TypeError: Cannot assign to read only property 'toJSON' of object '[object Object]'
    at Object.<anonymous> (/usr/lib/node_modules/mongoose/lib/error/validation.js:74:34)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/lib/node_modules/mongoose/lib/error/index.js:102:33)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
```

**If the current behavior is a bug, please provide the steps to reproduce.**

```
var kurento = require("kurento-client");
var mongoose = require("mongoose");
```

**What are the versions of Node.js, Mongoose and MongoDB you are using? Note that "latest" is not a version.**

I'm using version 3.3.1 for mongodb and 5.6.11 for mongoose
```
├─┬ mongodb@3.3.1 -> /usr/lib/node_modules/mongodb
├─┬ mongoose@5.6.11 -> /usr/lib/node_modules/mongoose
```

**Comment by vkarpov15** [comment]
This is a duplicate of https://github.com/mongodb/node-mongodb-native/pull/1293#issuecomment-166007944 - the `error-tojson` package by default makes it impossible to define custom `toJSON()` on error objects. See https://github.com/mongodb/node-mongodb-native/pull/1293#issuecomment-166007944.

TLDR, this will work fine if you swap the order of imports:

```javascript
var mongoose = require("mongoose");
var kurento = require("kurento-client");
```
